### PR TITLE
Add another species title

### PR DESCRIPTION
**Pull request by Implojin**
Ashenzari warns you: it is.

### CODE REVIEW

**Comment by sdynet** [comment]
I need this right now!

File: crawl-ref/source/skills.cc [hunk 1]
```diff
@@ -1688,13 +1688,14 @@ string skill_title_by_rank(skill_type best_skill, uint8_t skill_rank,
 
         case SK_UNARMED_COMBAT:
             if (species == SP_FELID)
-            {
                 result = claw_and_tooth_titles[skill_rank];
-                break;
+            else if (species == SP_MUMMY && skill_rank == 5 && god == GOD_NO_GOD)
```
Comment (ebering) [inline crawl-ref/source/skills.cc:1692]: It breaks with the meme but remove the god check and I'll merge this.

**Review by ebering — CHANGES_REQUESTED** [review]

File: crawl-ref/source/skills.cc (hunk 1 above)
Comment (Implojin) [reply crawl-ref/source/skills.cc:1692]: Yeah, that's fair.

(Despite the fact that I want to grumble about it~)

Comment (Implojin) [reply crawl-ref/source/skills.cc]: Okay, I've removed the god check and rebased to master.

**Review by ebering — APPROVED** [review]

**Comment by ebering** [comment]
Another satisfied customer

```
[12:27 PM] dilly: URHH who added pharaoh title
[12:27 PM] dilly: it just gives me only pain
```

**Pull request merged**